network/network.go

```diff
@@ -376,7 +376,10 @@ func (n *network) AllowConnection(nodeID ids.ShortID) bool {
 }
 
 func (n *network) Track(ip utils.IPCertDesc) {
-	nodeID := peer.CertToID(ip.Cert)
+	nodeID, err := peer.CertToID(ip.Cert)
+	if err != nil {
+		return
+	}
```

Comment:
this shadows the error from the fucntion call, will that be an issue? or can that be safely ignored?